workflow/util/util.go

```diff
@@ -600,7 +600,7 @@ func randString(n int) string {
 
 // RandSuffix generates a random suffix suitable for suffixing resource name.
 func RandSuffix() string {
-	return randString(5)
+	return randString(10)
```

Comment:
This will break some of the existing workflows where some names (workflow pods) might exceed the maximum length.

Reply:
would u merge if I put a if check on the max length?

Reply:
There is already a check on the length. The problem with this change is that people's workflows will break unexpectedly if the length is already close to the limit.